sdk/communication/Azure.Communication.CallAutomation/tests/CallMedias/CallMediaAutomatedLiveTests.cs

```diff
@@ -42,7 +42,8 @@ public async Task ContinuousRecognitionTest()
             var uniqueId = await ServiceBusWithNewCall(user, target);
 
             // create call and assert response
-            CreateCallResult response = await client.CreateCallAsync(new CallInvite(target), new Uri(TestEnvironment.DispatcherCallback + $"?q={uniqueId}"));
+            CallInvite invite = new CallInvite(target);
```

Comment:
If possible, maybe it would be better to use PhoneNumber ids for the dtmf recognition tests too?

Reply:
done, thanks!